<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmcrm</class>
 <widget class="QDialog" name="frmcrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>694</width>
    <height>491</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pgDolasci">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>760</width>
              <height>294</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Dolasci</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="btnPridruziRacun">
                 <property name="text">
                  <string>Pridruzi ovaj racun nakon unosa</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/ikone/ikone/list_add.svg</normaloff>:/ikone/ikone/list_add.svg</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Opis</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="txtDolasciOpis"/>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="minimumSize">
                    <size>
                     <width>75</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>75</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Prodavac</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="txtProdavac">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnProdavacTrazi">
                   <property name="minimumSize">
                    <size>
                     <width>25</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>25</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>V</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="txtProdavacNaziv">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="btnUpisIzmijena">
                 <property name="text">
                  <string>Upis</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/ikone/ikone/button_ok.svg</normaloff>:/ikone/ikone/button_ok.svg</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnIzlaz">
                 <property name="text">
                  <string>Izlaz</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/ikone/ikone/button_cancel.svg</normaloff>:/ikone/ikone/button_cancel.svg</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pgPregledDolazaka">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>760</width>
              <height>294</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Pregled Dolazaka</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="pregledDolazakaVerticalLayout">
               <item>
                <widget class="QPushButton" name="btnUcitajPregled">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 255, 127);</string>
                 </property>
                 <property name="text">
                  <string>Ucitaj preglede</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/ikone/ikone/document_preview.svg</normaloff>:/ikone/ikone/document_preview.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horLayoutPregledGumbi">
                 <item>
                  <widget class="QPushButton" name="btnPregBrisiCRM">
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 0);</string>
                   </property>
                   <property name="text">
                    <string>Brisi CRM</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnPregDodajVezuRacun">
                   <property name="text">
                    <string>Pridruzi na racun u izradi</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnPregMakniVezuRacun">
                   <property name="text">
                    <string>Makni vezu sa racuna u izradi</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTableView" name="dgPregledDolazaka"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pgPregDok">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>166</width>
              <height>133</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Pregled Dokumenata</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QPushButton" name="btnUcitajPregDok">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 255, 127);</string>
                 </property>
                 <property name="text">
                  <string>Ucitaj Preglede</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/ikone/ikone/document_preview.svg</normaloff>:/ikone/ikone/document_preview.svg</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableView" name="dgPregDok"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Kupac</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtKupac">
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnKupacTrazi">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtKupacNaziv">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>F2-Pozivanje</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
